### ISSUE TITLE
Pressing login does not dismiss keyboard and can obscure error modal

### ISSUE THREAD

**Issue by kelseybrennan12** [etn-ccis/blui-react-native-workflows#514]
#### Describe the bug / expected behavior

If a user has not dismissed the keyboard when they tap the login button, the keyboard remains on the screen and can obscure the error modal.

I would expect that pressing the login button should dismiss the keyboard.

#### What are the steps to reproduce?

1. Go to login screen on a real phone or a simulator with hardware keyboard disabled
2. Type in an invalid username/password, leave the keyboard open
3. Tap the "Log In" button

#### Screenshots / Screen recording

https://github.com/user-attachments/assets/48d37c5b-9de7-4070-bfcc-13033e69198b

#### Your environment information

- @brightlayer-ui/react-native-auth-workflow 6.0.1
- Reproducible on iOS and Android

**Comment by shubham-eaton** [comment]
We can try using react native keyboard object to dismiss the keyboard declaratively. Please refer the link 
https://reactnative.dev/docs/keyboard#dismiss

**Comment by kelseybrennan12** [comment]
@shubham-eaton We tried adding `onLogin={Keyboard.dismiss}` to the BLUI `LoginScreen` component but it didn't work; the keyboard is still displayed.

**Comment by shubham-eaton** [comment]
@kelseybrennan12  We are looking into it on priority.

**Comment by kelseybrennan12** [comment]
@shubham-eaton I tried upgrading from 6.0.1 to 7.0.1-alpha.0. When I opened the app, it crashed with this error:

```
error: Error: Unable to resolve module @okta/okta-react-native from /Users/kelsey/em-installer-mobile-app/node_modules/@brightlayer-ui/react-native-auth-workflow/lib/commonjs/screens/OktaRedirectLoginScreen/OktaRedirectLoginScreen.js: @okta/okta-react-native could not be found within the project or in these directories:
  node_modules
```

Are there additional upgrade steps that I need to take? I couldn't find anything in the docs.

**Comment by kelseybrennan12** [comment]
@shubham-eaton Can you confirm whether the error I shared above is a bug with the library or if there are additional upgrade steps I need to follow?

**Comment by shubham-eaton** [comment]
@kelseybrennan12  for this you need to install @okta/okta-react-native in your project's dependency and peer dependency, since v7 is introduced to support okta authentication , this package is necessary to install.

**Comment by shubham-eaton** [comment]
@kelseybrennan12 we are not planned yet to release a stable version yet.

**Comment by kelseybrennan12** [comment]
@shubham-eaton We need to pull this fix into the AbleEdge Installer App, and I would prefer to do it via a stable release. Would it be possible to get one in the next few weeks?

**Comment by shubham-eaton** [comment]
@kelseybrennan12  we have released the stable version v7.0.1.
